torch/onnx/_internal/fx/custom_operator.py

```diff
@@ -0,0 +1,58 @@
+from typing import Callable
+
+import torch
+from .function_dispatcher import _ATENLIB_FUNCTIONS, _OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
```

Comment:
Prefer full pathname import and module import only. See https://google.github.io/styleguide/pyguide.html#22-imports and https://google.github.io/styleguide/pyguide.html#23-packages
```suggestion
from torch.onnx._internal.fx import function_dispatcher
```
then use
`function_dispatcher._ATENLIB_FUNCTIONS`